src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link
+  href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+  rel="stylesheet"
+  >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        class="card__image"
+        alt="imac"
+      >
+      <div class="card__item-name">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
Using semantic tags in HTML, such as `<p>`, `<h1>`, `<section>`, and `<article>` etc, is important for several reasons:
1. Accessibility: Screen readers and assistive technologies rely on semantic tags to interpret content and provide a better experience for users with disabilities.
2. Search Engine Optimization (SEO): Search engines use semantic tags to understand the structure and content of a webpage. Properly using these tags can improve a website's search ranking and visibility.
3. Maintainability: Semantic tags make the code easier to understand and maintain by clearly indicating the purpose of each element. This benefits both the developer and any future collaborators.
4. Cross-platform compatibility: Semantic tags help ensure that content is displayed consistently across different devices and browsers, providing a better user experience.
5. Structured data: Semantic tags create a well-defined structure that can be easily parsed by machines and used for data analysis or other purposes.